manifests/e/Ergosoft/ErgoFAKT_V5/4.9.2.5294/Ergosoft.ErgoFAKT_V5.locale.de-DE.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Ergosoft.ErgoFAKT_V5
+PackageVersion: 4.9.2.5294
+PackageLocale: de-DE
+Publisher: ERGO SOFT Softwareentwicklung GmbH
+PublisherUrl: https://www.ergofakt.de/
+PackageName: ErgoFAKT V4.9.2.5294

```

Comment:
```suggestion
PackageName: ErgoFAKT
```

Reply:
Made changes in PackageName and used "wingetcreate submit .\" to "commit".

Seems that this generated a second pull request.
Should I remove this (now older) one (and how)?

Reply:
Unfortunately, the version in the `PackageName` needs to be removed as this will make updating manifests a bit more tedious.

You can make the changes in your branch using the web version of GitHub or the client on Windows.

Reply:
Changed PackageName to "ErgoFAKT V5" to be able to provida a future version as new product eg. "ErgoFAKT V6"